packages/formik/src/types.tsx

```diff
@@ -113,8 +117,7 @@ export interface FormikHelpers<Values> {
   setFormikState: (
     f:
       | FormikState<Values>
-      | ((prevState: FormikState<Values>) => FormikState<Values>),
-    cb?: () => void
```

Comment:
this may be the only breaking change in the API (but it's not used anymore). Maybe we can mark it as `deprecated`?

Reply:
I think you're right, seems old. We can add `@deprecated` here.

Reply:
Perfect! It seems that JSDocs can't annotate a param as deprecated, only values.

I changed the name from cb to cbDeprecated to be blunt about it and add an inline deprecation that I read that typescript interprets (but it's not working on vscode).

If you have a better alternative, please tell me :)